monitoring/mock_uss/config.py

```diff
@@ -40,3 +40,6 @@ class Config(object):
     DSS_URL = os.environ.get(ENV_KEY_DSS, None)
     BEHAVIOR_LOCALITY = Locality(os.environ.get(ENV_KEY_BEHAVIOR_LOCALITY, "CHE"))
     CODE_VERSION = os.environ.get(KEY_CODE_VERSION, "Unknown")
+    CERT_PATH = "/var/test-certs"
+    PRIVATE_KEY_PATH = "{}/messagesigning/mock_faa_priv.pem".format(CERT_PATH)
```

Comment:
Since we want to [eventually be able to change these paths at runtime](https://github.com/interuss/dss/tree/master/monitoring/uss_qualifier/suites/faa/uft/design#negative-tests--), these shouldn't be items set in the static configuration of mock_uss.  Until we add the infrastructure to change these paths dynamically, I think it's fine to hardcode a path value in the handler with a TODO note pointing to the documentation saying what will happen eventually.

I would expect that eventually the `msgsigning` service capability would acquire its own "database" (like all the other service capabilities have) and one of the "tables" would be a setting of which key pair to use.